tests/suite-unit/PluginFlyvemdmTask.php

```diff
@@ -148,8 +169,8 @@ public function providerPrepareInputForAdd() {
       ]);
 
       $existingPolicyId = $existingPolicy->getID();
-      $nonExistingPolicyId = \PluginFlyvemdmCommon::getMax($existingPolicy, '', 'id') + 1;
-      $nonExistingFleetId = $fleet->getID() + 1;
+      $nonExistingPolicyId = -1;
+      $nonExistingFleetId = -1;
```

Comment:
I disagree to use -1. The purpose of these statements is to check how the code behaves when an instance was deleted. We must use a probable value, which is in the interval [1, 99999999999] (int(11))